frontend/astro.config.mjs

```diff
@@ -12,14 +12,14 @@ export default defineConfig({
       aposHost: 'http://localhost:3000',
       widgetsMapping: './src/widgets',
       templatesMapping: './src/templates',
-      forwardHeaders: [
-        'content-security-policy', 
-        'strict-transport-security', 
+      includeResponseHeaders: [
+        'content-security-policy',
+        'strict-transport-security',
         'x-frame-options',
         'referrer-policy',
         'cache-control'
         // Should only be passed in multisite projects, and only if Apostrophe and Astro are hosted together
-        // 'host'
+        'host'
```

Comment:
This header does not exist at all in the response direction. So just remove it.

Reply:
We need

```
excludeRequestHeaders: [
  // Must exclude this for separate apostrophe and astro hosting to work
  'host'
```

Let's not even mention multisite here. We should work up a separate `combined-astro-assembly-starter-kit` that does not exclude this header, and documents the need to host them together, share certificates or use our hosting. Which is a conversation we will only be having with paying customers so we can help them in more detail.

Reply:
I missed that you had asked for changes - no email notification. Since we are on Apostrophe hosting, shouldn't this header still be passed? So, it should be moved but commented out. I've been away from the keyboard too long already. Ahhh!